src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTape.java

```diff
@@ -171,84 +157,92 @@ public IBlockState getStateFromMeta(final int meta)
         return this.getDefaultState().withProperty(FACING, enumfacing);
     }
 
-    /**
-     * @deprecated (Remove this as soon as minecraft offers anything better).
-     */
-    @SideOnly(Side.CLIENT)
     @Override
-    @Deprecated
-    public boolean shouldSideBeRendered(IBlockState blockState, IBlockAccess blockAccess, BlockPos pos, EnumFacing side)
+    public int getMetaFromState(@NotNull final IBlockState state)
     {
-        return true;
+        return state.getValue(FACING).getIndex();
     }
 
     /**
      * @deprecated (Remove this as soon as minecraft offers anything better).
      */
+    @SuppressWarnings(DEPRECATION)
     @Override
     @Deprecated
-    public boolean isOpaqueCube(final IBlockState state)
+    public boolean isFullCube(final IBlockState state)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)